### DISCUSSION TITLE
Please support Copilot proxy

### DISCUSSION THREAD

**Discussion by roojay** [community/community#40931] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Since today afternoon getting an error: github copilot could not connect to server. extension activation failed: "read econnreset"

So I tested the access to github in China, em......
![image](https://user-images.githubusercontent.com/11495164/205932406-a111ecae-b3a9-4afc-a02e-2c83b60fa1ad.png)

**Comment by weiweirj** [comment]
Me too.

**Comment by iamcjwilliams** [comment]
@roojay520 Sorry for the trouble here, Could you provide a few more details on how this can be replicated? Also please perform the following (View -> Output; select GitHub Copilot from the pulldown)

**Reply by roojay** [reply]
GitHub Copilot could not connect to server. Extension activation failed: "read ECONNRESET"

**Comment by roojay** [comment, marked as answer]
If use vscode:

Settings > Search settings search "proxy" > Set up your local proxy

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/11495164/206605533-2c4d459d-49aa-4bfa-8895-e42773107ac8.png">

Reference: [Something Initial HTTP proxy support for Copilo](https://github.com/orgs/community/discussions/29127)
